src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -163,6 +165,14 @@ private void handleExit() {
         primaryStage.hide();
     }
 
+    /**
+     * Opens up FavouriteWindow by setting up the fxml scene and opening it
+     */
+    @FXML
+    private void handleFavourite(ActionEvent event) {
+        favouriteWindow.show();
+    }
```

Comment:
Perhaps you might want to add checking for if the favorite window is open already? Maybe something similar to what is being done for the help window
```
   @FXML
    private void handleFavourite(ActionEvent event) {
        if (!favoriteWindow.isShowing()) {
            favouriteWindow.show();
        } else {
            favouriteWindow.focus();
        }
    }
```

Reply:
Thanks @howardwhw2, I have made the necessary changes. By the way, I am still trying to solve the CI workflow because all my tests passed locally, but it does not on Github and I am not sure why.

Reply:
Hmm that's strange, did you remove/edit any of the unit tests locally?

If not, could you try running that 1 specific unit test your feature is failing? `execute_allFieldsSpecifiedUnfilteredList_success()` Test of `EditCommandTest`

Reply:
I tested that specific unit test and it passed
![image](https://user-images.githubusercontent.com/56862300/157578388-46e0a1ee-b182-4984-a185-d3fdc5f503be.png)

Reply:
any chance your tests were not set up properly? you could perhaps run through the setup again on the cs2103 website